main.go

```diff
@@ -24,7 +24,7 @@ import (
 
 const (
 	defaultLogFilePath = "/host/var/log/aws-routed-eni/ipamd.log"
-	version            = "0.1"
+	version            = "1.0.1"
```

Comment:
We shouldn't say 1.y.z while still in pre-production state; this should be 0.1.1 (patch change from prior).

From [Semver standard](https://semver.org/):
4. Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.
5. Version 1.0.0 defines the public API. The way in which the version number is incremented after this release is dependent on this public API and how it changes.